sys/fscontrol.c

```diff
@@ -380,7 +378,26 @@ DokanDispatchFileSystemControl(__in PDEVICE_OBJECT DeviceObject,
       vpb = irpSp->Parameters.MountVolume.Vpb;
       DokanInitVpb(vpb, dcb->DeviceObject, vcb->DeviceObject);
 
-      // FsRtlNotifyVolumeEvent(, FSRTL_VOLUME_MOUNT);
+	  
+	  PFILE_OBJECT VolumeFileObject = NULL;
+	  status = STATUS_SUCCESS;
+	  try {
+		  VolumeFileObject = IoCreateStreamFileObjectLite(NULL, vcb->DeviceObject);
+	  } except(EXCEPTION_EXECUTE_HANDLER) {
+		  status = GetExceptionCode();
+		  DDbgPrint("   IoCreateStreamFileObjectLite failed with status code %d\n", status);
+	  }
+
+	  if (status == STATUS_SUCCESS) {
+		  // notify all application about volume mount
+		  FsRtlNotifyVolumeEvent(VolumeFileObject, FSRTL_VOLUME_MOUNT);
```

Comment:
Wow, I thought this was automatically done by I/O manager once we returned success from IRP_MN_MOUNT_VOLUME. Looks like it is needed indeed (see https://github.com/dokan-dev/dokany/pull/140/files#r49822003).